core/market/src/negotiation/provider.rs

```diff
@@ -51,6 +52,7 @@ impl ProviderBroker {
         let broker3 = broker.clone();
         let broker_proposal_reject = broker.clone();
         let broker_terminated = broker.clone();
+        let commit_broker = broker.clone();
```

Comment:
I have in some old branch, a builder for Brokers, that would allow to get rid of this, but I'm to shy (read: it is to close to Mainnet release) to reveal it now ;)